test/_external/test262/skiplist

```diff
@@ -1,3 +1,9 @@
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-rqstd-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-self-once.js
+
```

Comment:
They use a test harness method called `fnGlobalObject` 👆

Reply:
I believe `fnGlobalObject` should be injected. I think the test failure was saying that the module was being loaded several times. let me check.

Reply:
In my local test of `eval-self-once` (replacing the global helper with `Function("return this")()`, turning the `assert` into a `console.log` and commenting out the `.then($DONE,$DONE)` I get

```
1 1 'global property was defined and incremented only once'
```

Reply:
We have tests for eval count ourselves too which is why I'm keying in on this skip as something odd.

Reply:
aaah. sorry. I think I saw something with `$DONE` being `undefined`

Reply:
I'll have another look.

Reply:
I must have missed that: 
https://github.com/tc39/test262/blob/master/harness/doneprintHandle.js

thanks for pointing it out!

I didn't look into it further at that time as I wanted the test run to be green first - and then investigate. so there's a chance that some other tests are failing because of that or similar issues as well.

Reply:
That's cool. (We have tests covering this so it's less critical).

Reply:
I looked into the failure. the reason was that the requested-once-`import` and requested-once-`dynamic import` tests are using the same global variable `global.test262`. the `import` tests would set it (and don't remove it afterwards), which makes the `dynamic import` tests fail.

I'll remove the `global.test262` for now in every it(...). not sure if every test is supposed to run in a fresh and clean context.